lightly/api/api_workflow_export.py

```diff
@@ -314,23 +316,29 @@ def export_filenames_and_read_urls_by_tag_id(
             ]
 
         """
-        filenames_string = retry(
-            self._tags_api.export_tag_to_basic_filenames,
-            dataset_id=self.dataset_id,
-            tag_id=tag_id,
-            file_name_format=FileNameFormat.NAME,
+        filenames_string = list(
```

Comment:
@MalteEbner I am not sure if this is 100% python safe as get strings back here and not arrays. Could you doublecheck this with the paginate_endpoint iterator please?

Reply:
This was not correct. I think I fixed it with my commits but I'll have to test it properly to be sure.